docs/conf.py

```diff
@@ -0,0 +1,29 @@
+"""Configuration file for the Sphinx documentation builder."""
+
+external_projects_remote_repository = "https://raw.githubusercontent.com/ROCm/dcgpu-docs/refs/heads/develop/docs/projects.yaml"
+external_projects_current_project = "device-metrics-exporter"
+
+project = "AMD Instinct Documentation"
+version = "1.0.0"
+release = version
+html_title = f"Device Metrics Exporter {version}"
+author = "Advanced Micro Devices, Inc."
+copyright = "Copyright (c) 2024 Advanced Micro Devices, Inc. All rights reserved."
+
+# Required settings
+html_theme = "rocm_docs_theme"
+html_theme_options = {
+    "flavor": "generic",
+    "header_title": "AMD Instinct Documentation",
+    "nav_secondary_items": {
+        "GitHub": "https://github.com/rocm/dcgpu-docs",
```

Comment:
Do we need to change the Github Link here to `https://github.com/rocm/device-metrics-exporter`?